### ISSUE TITLE
low e2e precision

### ISSUE THREAD

**Issue by azade-a** [PaddlePaddle/PaddleOCR#4446]
Hello.
I used e2e_r50_vd_pg.yml config( just change lr to 0.0001 and use Cosine schedule_lr) and trained PGNet on total_text dataset based on start training section (https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.1/doc/doc_en/pgnet_en.md#start-training).
Unfortunately I got this low e2e precision:
root INFO: best metric, f_score_e2e: 0.45739166493883476, total_num_gt: 2543, total_num_det: 2280, global_accumulative_recall: 1729.7999999999954, hit_str_count: 1103, recall: 0.6802202123476191, precision: 0.7685087719298228, f_score: 0.7216742478495215, seqerr: 0.36235402936755523, recall_e2e: 0.4337396775462053, precision_e2e: 0.4837719298245614, fps: 20.294347350078876, best_epoch: 475
I appreciate your help in advance.

**Comment by tampe125** [comment]
@LDOUBLEV any chances to have the trained model after "step2", too?

**Comment by chaowang1994** [comment]
> Emm, the pre-trained model is needed for training total-text.
> 
> And there are bugs in the end-to-end indicator function, which will result in low evaluation accuracy. We will fix it soonly.
> 
> And the learning rate 0.0001 is too small for training

Is this bug fixed?

**Comment by chaowang1994** [comment]
@LDOUBLEV

**Comment by tampe125** [comment]
For what it's worth, I just tried with PaddlePaddle 2.2.0 with cuda 11.2 and the problem is still there, accuracy is still terrible (35%)
